overrides/engine/prefabs/defaultVisualCharacter.prefab

```diff
@@ -0,0 +1,16 @@
+{
+    "SkeletalMesh" : {
+        "mesh" : "lightAndShadowResources:pawn",
+        "heightOffset" : -0.8,
+        "material" : "lightAndShadowResources:whitePawnPlayerSkin",
+        "animation" : "lightAndShadowResources:pawnStill",
+        "loop" : true
+    },
+    "Stand" :{
+        "animationPool": ["lightAndShadowResources:pawnStill"]
```

Comment:
I'm not sure why the code has `LightAndShadowResources` while the prefab has `lightAndShadowResources` - if it's not intentional, it might cause problems in *nix systems that have case sensitive paths. If it is intentional, please ignore my comment - otherwise have a look.